pkg/controller/node/nodecontroller.go

```diff
@@ -341,6 +348,9 @@ func NewNodeController(
 		}
 	}
 
+	// Register prometheus metrics
+	Register()
```

Comment:
I think we generally want to register metrics in init() function (instead of here) - that is the patern that we are using in the code.